sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultJcaProvider.java

```diff
@@ -34,6 +39,26 @@
      */
     private static final Double VERSION = 1.0;
 
+    /**
+     * The default algorithm for certificate sign which Key Type is RSA in key Vault will be used
```

Comment:
How about defining 
```
public abstract String getAlgorithmName()
``` 
in `AbstractKeyVaultKeyLessSignature` and implement it in it sub classes?